src/system_identity/output_platform.rs

```diff
@@ -0,0 +1,9 @@
+use std::path::PathBuf;
+
+/// This type represents the output platform for the authentication credentials created with
```

Comment:
why the `platform` term? isn't this a SI authentication method for the generated SI?

Reply:
it could be an input as well right? , we could end up with both types representing the same data ( a SI auth configuration) for input as well as for output

Reply:
Well the idea I had is that this "output platform" is an output of the program itself (e.g. it creates and persists the key pair, using the path for doing so) but at the same time its configuration (the actual path where they will be persisted) is an input, the caller of the CLI specifies this path, so the value is constructed as an input and is used in the `SystemIdentityCreationMetadata`.

Reply:
>Well the idea I had is that this "output platform" is an output of the program itself (e.g. it creates and persists the key pair, using the path for doing so) but at the same time its configuration (the actual path where they will be persisted) is an input, the caller of the CLI specifies this path, so the value is constructed as an input and is used in the SystemIdentityCreationMetadata.

If is used as input as well as output  would make sense to call it AuthMethod (or something similar)  and then perhaps implement  output methods for printing the values as they are expected, perhaps `Display` or some special serialization ?

Reply:
But it's not an auth method actually no? I mean it's only where the credentials are persisted or "emitted". One possible variant of this enum could be something like `PrintToStdout` and then when the SI is generated, when this value is inspected it might call `println!` for the structure or something like this. A more fitting name could be what @danielorihuela suggests. WDYT?